src/main/java/org/springframework/data/tarantool/core/mappers/TarantoolAutoResultConverter.java

```diff
@@ -0,0 +1,73 @@
+package org.springframework.data.tarantool.core.mappers;
+
+import io.tarantool.driver.api.TarantoolResult;
+import io.tarantool.driver.api.metadata.TarantoolSpaceMetadata;
+import io.tarantool.driver.core.TarantoolResultImpl;
+import io.tarantool.driver.mappers.ValueConverter;
+import org.msgpack.value.ArrayValue;
+import org.msgpack.value.StringValue;
+import org.msgpack.value.Value;
+import org.msgpack.value.ValueFactory;
+
+import java.util.Map;
+
+/**
+ * Converter allows you to check if the received result is suitable for converting to {@link TarantoolResult},
+ * i.e. is the response structure appropriate and is the metadata present
+ *
+ * @param <V> the source MessagePack entity type
+ * @param <T> the target object type
+ * @author Artyom Dubinin
+ */
+public class TarantoolAutoResultConverter<V extends Value, T> implements ValueConverter<V, TarantoolResult<T>> {
```

Comment:
Actually it is a "CRUD" converter. I think we need to have a special type for that and the corresponding naming.

Reply:
A link in the description to the tarantool/crud README will be also necessary.

Reply:
In general, this converter is a repeat of this `TarantoolResultConverter`, only `canConvertValue` is added. Therefore, I redid it a little and now the class inherits from `TarantoolResultConverter`. It seems to me that the abbreviation `CRUD` is overloaded. 

@wey1and @vrogach2020 can you suggest a better option?

Reply:
With @vrogach2020 we decided that because we are left with the following API:
```
AUTO - default
TUPLE
```
then this converter can be left as it is now named - `TarantoolAutoResultConverter`